mux/src/tab.rs

```diff
@@ -751,6 +751,35 @@ impl Tab {
     pub fn get_zoomed_pane(&self) -> Option<Arc<dyn Pane>> {
         self.inner.lock().get_zoomed_pane()
     }
+
+    pub fn save(&self) -> Option<SavedTab> {
+        fn convert_node(t: &Tree) -> Option<SavedTree> {
```

Comment:
can we leverage/reuse some or all of the `pane_tree` and `build_from_pane_tree` logic in this file that is used by the mux protocol here?